pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -0,0 +1,23 @@
+package bootstrap
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
+func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
+	errorList := field.ErrorList{}
+	if installConfig.BootstrapInPlace == nil {
+		return field.Required(field.NewPath("BootstrapInPlace"), "missing BootstrapInPlace configuration")
+	}
+	if *installConfig.ControlPlane.Replicas != 1 {
+		errorList = append(errorList, field.Invalid(field.NewPath("ControlPlane").Child("Replicas"), installConfig.ControlPlane.Replicas,
+			"bootstrap in place requires a single ControlPlane replica"))
+	}
+	if installConfig.BootstrapInPlace.InstallationDisk == "" {
```

Comment:
Is there any other validation that we may want to perform on the installation disk?